hack/csv-generate.sh

```diff
@@ -69,19 +71,19 @@ if [[ "$CSV_VERSION" != "$PREV" ]]; then
     --csv-version "${CSV_VERSION}" \
     --operator-csv-template-file "${CSV_FILE}" \
     --operator-image "${FULL_OPERATOR_IMAGE}" \
-    --olm-bundle-directory "$OUT_CSV_DIR" \
-    --replaces-csv-version "$REPLACES_CSV_VERSION" \
-    --skip-range "$CSV_SKIP_RANGE" \
+    --olm-bundle-directory "${OUT_CSV_DIR}" \
+    --replaces-csv-version "${REPLACES_CSV_VERSION}" \
```

Comment:
Do we need this? Now that we use skipRange?

Reply:
removed